domains/auth/Auth.API/API/Controllers/AuthController.cs

```diff
@@ -2,20 +2,27 @@
 using API.Services;
 using Microsoft.AspNetCore.Mvc;
 using System.ComponentModel.DataAnnotations;
+using API.Configuration;
+using Microsoft.Extensions.Options;
 
 namespace API.Controllers;
 
 
 [ApiController]
-public class AuthController
+public class AuthController : ControllerBase
 {
     readonly ILogger<AuthController> logger;
     readonly IOidcProviders oidcProviders;
+    readonly ICookieService cookieService;
```

Comment:
Rename these to have an underscore(_) in front of their name. 
This is due to they're only used in this class